apps/cyberstorm-storybook/public/css/globals.css

```diff
@@ -108,22 +109,48 @@
   --color-gradient-purple-green--dark: linear-gradient(to left top, #2f0e44 0%, #0a4c35 100%);
   --color-gradient-purple-green: linear-gradient(to left top, #9d30e4 0%, #23ffb0 100%);
   --color-gradient-purple-green--light: linear-gradient(to left top, #b159e9 0%, #73e1b2 100%);
-  --size-tiny: 0.25em;
-  --size-xxs: 0.375em;
-  --size-xs: 0.5em;
-  --size-s: 0.625em;
-  --size-m: 1em;
-  --size-l: 1.25em;
-  --text-xxxs: 0.5em;
-  --text-xxs: 0.625em;
-  --text-xs: 0.75em;
-  --text-s: 0.875em;
-  --text-m: 1em;
-  --text-l: 1.125em;
-  --text-xl: 1.25em;
-  --text-xxl: 1.375em;
-  --text-xxxl: 1.5em;
-  --gap-m: var(--size-m);
-  --padding-m: var(--size-m);
-  --padding-s: var(--size-s);
+  --font-weight-semi-bold: 600;
+  --font-weight-bold: 700;
+  --size--xxxxxxs: 0.125em;
+  --size--xxxxxs: 0.25em;
+  --size--xxxxs: 0.375em;
+  --size--xxxs: 0.5em;
+  --size--xxs: 0.625em;
+  --size--xs: 0.75em;
+  --size--s: 0.875em;
+  --size--m: 1em;
+  --size--l: 1.125em;
```

Comment:
Why use double hyphen between `size` and `l` but single hyphen in `--border-radius-l`, `gap-m` etc?

Reply:
I dunno, used some logic which doesn't seem smart anymore. Changed to single hyphens.